src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateSDOCallbackHandler.java

```diff
@@ -513,22 +529,29 @@ private CallbackResponse prePopulateOrderDetailsPages(CallbackParams callbackPar
         SmallClaimsHearing tempSmallClaimsHearing = SmallClaimsHearing.builder()
             .input1("The hearing of the claim will be on a date to be notified to you by a separate notification. "
                         + "The hearing will have a time estimate of")
-            .input2("The claimant must by no later than 14 days before the hearing date, pay the court the "
-                        + "required hearing fee or submit a fully completed application for Help with Fees. If the "
-                        + "claimant fails to pay the fee or obtain a fee exemption by that time the claim will be "
-                        + "struck without further order.")
+            .input2(featureToggleService.isHearingAndListingSDOEnabled() ? HEARING_TIME_TEXT_AFTER_HNL
+                        : HEARING_TIME_TEXT_AFTER)
             .build();
 
         updatedData.smallClaimsHearing(tempSmallClaimsHearing).build();
 
-        SmallClaimsNotes tempSmallClaimsNotes = SmallClaimsNotes.builder()
-            .input("This Order has been made without a hearing. Each party has the right to apply to have this Order "
-                       + "set aside or varied. Any such application must be received by the Court, "
-                       + "together with the appropriate fee by 4pm on")
-            .date(LocalDate.now().plusWeeks(1))
-            .build();
+        SmallClaimsNotes.SmallClaimsNotesBuilder tempSmallClaimsNotes = SmallClaimsNotes.builder();
+        if (featureToggleService.isHearingAndListingSDOEnabled()) {
+            tempSmallClaimsNotes.input("Each party has the right to apply to have this Order set aside or varied. "
+                    + "Any such application must be received by the Court "
+                    + "(together with the appropriate fee) by 4pm on "
+                    + DateFormatHelper.formatLocalDate(
+                    deadlinesCalculator.plusWorkingDays(LocalDate.now(), 5), DATE)
+            );
+        } else {
+            tempSmallClaimsNotes.input(
+                    "This Order has been made without a hearing. Each party has the right to apply to have this Order "
+                        + "set aside or varied. Any such application must be received by the Court, "
+                        + "together with the appropriate fee by 4pm on")
+                .date(LocalDate.now().plusWeeks(1));
+        }
 
-        updatedData.smallClaimsNotes(tempSmallClaimsNotes).build();
```

Comment:
Not really your change so feel free to ignore but I don't think there is any need to do the overall build here?